packages/playground/blueprints/src/lib/steps/run-sql.ts

```diff
@@ -0,0 +1,90 @@
+import { StepHandler } from '.';
+import { rm } from './rm';
+
+/**
+ * @inheritDoc runSql
+ * @hasRunnableExample
+ * @example
+ *
+ * <code>
+ * {
+ *		"step": "runSql",
+ *		"sql": {
+ *			"resource": "literal",
+ *			"name": "schema.sql",
+ *			"contents": "SELECT * FROM wp_posts"
+ *		},
+ * }
+ * </code>
+ */
+export interface RunSqlStep<ResourceType> {
+	/**
+	 * The step identifier.
+	 */
+	step: 'runSql';
+	/**
+	 * The SQL to run.
+	 */
+	sql: ResourceType;
+	/**
+	 * Whether to stop on errors
+	 */
+	stopOnErrors?: boolean;
+}
+
+/**
+ * Run an SQL file.
+ */
+export const runSql: StepHandler<RunSqlStep<File>> = async (
+	playground,
+	{ sql, stopOnErrors = true },
+	progress?
+) => {
+	progress?.tracker.setCaption(`Executing SQL File`);
+
+	const sqlFilename = `/tmp/${crypto.randomUUID()}.sql`;
+
+	await playground.writeFile(
+		sqlFilename,
+		new Uint8Array(await sql.arrayBuffer())
+	);
+
+	const docroot = await playground.documentRoot;
+
+	const runPhp = await playground.run({
+		code: `<?php
+		require_once ${JSON.stringify(docroot)} . '/wp-load.php';
```

Comment:
We have a `phpVars` helper for bringing JS values into PHP code:

```ts
import { phpVars } from '@php-wasm/util';

const js = phpVars({
	docroot,
	sqlFilesname
});

const runPhp = await playground.run({
	code: `<?php
	require_once ${js.docroot} . '/wp-load.php';
	// ...
```

Reply:
Done: https://github.com/WordPress/wordpress-playground/pull/745/files#diff-ef7fc55c9e2c228ab627cd90d8372d926a0d8625f5f8fc76baf10cd15ed4f85fR51